test/repo-test.js

```diff
@@ -99,4 +99,32 @@ module.exports = (repo) => {
       })
     })
   })
+
+  describe('stat', () => {
+    it('get stats', (done) => {
+      repo.stat((err, stats) => {
+        expect(err).to.not.exist()
+        expect(stats).to.exist()
+        expect(stats).to.have.property('numObjects')
+        expect(stats).to.have.property('version')
+        expect(stats).to.have.property('repoPath')
+        expect(stats).to.have.property('repoSize')
+        expect(stats).to.have.property('storageMax')
+        done()
```

Comment:
just realized it now, these don’t test that they are the right numbers, so you should at least check thatj they are greater 0 or similar

Reply:
@dignifiedquire done! had to move the tests to another file so they run after the other tests add some data to the repo 😄